bike_rent/models/website.py

```diff
@@ -0,0 +1,11 @@
+from odoo import api, models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    @api.multi
+    def sale_product_domain(self):
+        result = super(Website, self).sale_product_domain()
+        result += [('bike_shop', '=', True), ('type', '=', 'service')]
```

Comment:
list object has a method for this, can you tell me what it is?

Reply:
Append or extend? I mentioned that to AndriusP when he asked how can we add items to the list, but he said to just use += here. I presume extend would work best here?